packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -122,17 +131,26 @@ function TableView<T extends object>(props: SpectrumTableProps<T>, ref: DOMRef<H
     }
   }, [scale]);
 
+  const getDefaultMinWidth = useCallback(({hideHeader, isSelectionCell, showDivider}) => {
+    if (hideHeader) {
+      let width = DEFAULT_HIDE_HEADER_CELL_WIDTH[scale];
+      return showDivider ? width + 1 : width;
+    } else if (isSelectionCell) {
+      return SELECTION_CELL_DEFAULT_WIDTH[scale];
+    }
+    return 75;
+  }, [scale]);
+
   let [isInResizeMode, setIsInResizeMode] = useState(false);
+  // entering resizing/exiting resizing doesn't trigger a render
+  // with table layout, so we need to track it here
+  let [, setIsResizing] = useState(false);
```

Comment:
will add to comment
```
  // Starts when the user selects resize from the menu, ends when resizing ends
  // used to control the visibility of the resizer Nubbin
  let [isInResizeMode, setIsInResizeMode] = useState(false);
  // Starts when the resizer is actually moved
  // entering resizing/exiting resizing doesn't trigger a render
  // with table layout, so we need to track it here
  let [, setIsResizing] = useState(false);
```

Reply:
What do you need to trigger when the resizer is moved though since the state isn't actually used? Wouldn't it also trigger a relayout?

Reply:
In this particular case, i really need the resize end, that's the important one. Without this, the blue line that stretches the entire tableview can get stuck in a visible mode if you drag off the table during resize and release out there. The final size is already done, so no more renders triggered to turn it off.

If we tracked the currently resizing column in state, that would do it, but since it's inside the columnlayout, it's not triggering a state update.

I could also call a relayout potentially. Though a quick test of that shows that the resizer for the body is hidden as expected, but the resizers in all the column headers stay visible

Reply:
ok, we can revisit this but feels weird. wouldn't it go out of resize mode at that point too?

Reply:
lets revisit
TODO: Add to followup